kibbeh/src/modules/room/chat/RoomChatList.tsx

```diff
@@ -147,12 +147,14 @@ export const RoomChatList: React.FC<ChatListProps> = ({ room }) => {
                                 );
                               case "emote":
                                 return emoteMap[v.toLowerCase()] ? (
-                                  <img
-                                    key={i}
-                                    className="inline"
-                                    alt={`:${v}:`}
-                                    src={emoteMap[v.toLowerCase()]}
-                                  />
+                                  <>
```

Comment:
Is this supposed to be closed like this?

Reply:
yeah remove this fragment